Misc/NEWS.d/next/Core and Builtins/2019-07-08-19-08-38.bpo-37367.SVtBhh.rst

```diff
@@ -0,0 +1,2 @@
+Byte strings containing octal values greater than 255 will now raise
+ValueError.  Patch by (Jeffrey Kintscher).
```

Comment:
Why is your name between parentheses?

Reply:
Again, I would use the word "escape sequences" here.

Reply:
> Why is your name between parentheses?

That's a typo.  I will remove it.